docs/run-conformance-tests.md

```diff
@@ -3,16 +3,75 @@ title: Conformance tests
 weight: 50
 ---
 
-This document enumerates the steps required to run conformance tests for various platforms supported by Lokomotive.
+This document enumerates the steps required to run conformance tests for various platforms supported
+by Lokomotive.
 
-**Note**: There is only one caveat to consider when running tests for AWS. For other platforms you can run conformance tests without making special arrangements.
+**Note**: Make the platform-specific changes before running the conformance tests.
 
 ## AWS
 
-For AWS you need to make sure that node ports are allowed in the security group. To do so make sure you set the `expose_nodeports` cluster property to `true` in the AWS config. Read more about this flag in the [AWS reference docs](configuration-reference/platforms/aws.md).
+For AWS you need to make sure that node ports are allowed in the security group. To do so, make sure
+you set the `expose_nodeports` cluster property to `true` in the AWS config. Read more about this
+flag in the [AWS reference docs](configuration-reference/platforms/aws.md).
 
-To install the cluster on AWS follow the [AWS quick start guide](quickstarts/aws.md).
+To install the AWS cluster, follow the [AWS quick start guide](quickstarts/aws.md).
+
+## Packet
+
+### Cluster Requirements
+
+Create a cluster with at least two worker nodes. To install the Packet cluster, follow the [Packet
+quick start guide](quickstarts/packet.md).
+
+### Expose kube-proxy
+
+Edit the kube-proxy Daemonset config to expose the metrics port on all interfaces. Run the following
+command to edit the configuration:
+
+```bash
+kubectl -n kube-system edit ds kube-proxy
+```
+
+Change the following flag from `- --metrics-bind-address=$(HOST_IP)` to
+`- --metrics-bind-address=0.0.0.0`.
+
+### Delete the mutating webhook server
+
+Run the following commands to delete the mutating webhook server that disallows the usage `default`
+service account tokens:
+
+```bash
+helm -n lokomotive-system uninstall lokomotive
+kubectl delete ns lokomotive-system
```

Comment:
Is it necessary to delete the namespace? Because what if `web-ui` is installed?

Reply:
If you want default SA to have `automountServiceAccountToken`. We could delete default SA after `locomotive` is deleted. Then the new SA created won't have patched SA

Reply:
- How is web-ui has to do anything with conformance tests?
- for the second comment, we are not concerned about the permissions of the service accounts in the `lokomotive-system` or `kube-system` but about the new namespaces conformance test creates to run the tests.

Reply:
I would just remove the webhook object. It will be less intrusive to the cluster then. And you can use helm to reinstall it later on.